cmd/builder.go

```diff
@@ -65,31 +65,66 @@ func NewCmdVersion(fullName string, versionInfo k8sversion.Info, buildahVersion
 
 // NewCommandS2IBuilder provides a CLI handler for S2I build type
 func NewCommandS2IBuilder(name string) *cobra.Command {
+	var isolation, ociRuntime, storageDriver, storageOptions string
+
 	cmd := &cobra.Command{
 		Use:   name,
 		Short: "Run a Source-to-Image build",
 		Long:  s2iBuilderLong,
 		Run: func(c *cobra.Command, args []string) {
-			err := cmd.RunS2IBuild(c.OutOrStderr())
+			var err error
+			if isolation == "" {
+				isolation, err = builderDefaultIsolation()
+				kcmdutil.CheckErr(err)
+			}
+			if storageDriver == "" {
+				storageDriver, storageOptions, err = builderDefaultStorage()
+				kcmdutil.CheckErr(err)
+			}
+			err = cmd.RunS2IBuild(c.OutOrStderr(), isolation, ociRuntime, storageDriver, storageOptions)
 			kcmdutil.CheckErr(err)
 		},
 	}
 
+	flags := cmd.Flags()
+	flags.StringVar(&isolation, "isolation", isolation, "type of process `isolation` to use for RUN instructions")
+	flags.StringVar(&ociRuntime, "oci-runtime", ociRuntime, "runtime to invoke for OCI isolation")
```

Comment:
What happens if `--oci-runtime` is not specified? Is there a reasonable default for the empty string?

Reply:
We can retrieve a default from `github.com/containers/common/pkg/config`'s `DefaultConfig()` function.  I'll add that.

Reply:
ok see that with https://github.com/openshift/builder/pull/220/files#diff-be76072c86dc24313ebc27eca7d613a1068dc337b068610f3f0e98f6e97ac5b0R71